doc/api/console.markdown

```diff
@@ -2,29 +2,79 @@
 
     Stability: 2 - Stable
 
-The module defines a `Console` class and exports a `console` object.
+The `console` module provides a simple debugging console that is similar, but
+not identical to the JavaScript console mechanism provided by Web browsers.
```

Comment:
Nit `is similar to the JavaScript console mechanism...` No need to say `but not identical` as that is what `similar` means.

If you *do* keep `but not identical1`, please add a comma after `identical`.